### DISCUSSION TITLE
Issue with TreeSelect selectionMode checkbox: unselecting a checkbox closes the overlay

### DISCUSSION THREAD

**Discussion by manu7823** [primefaces/community#459] [PrimeReact Blocks]
Hi, 

I'm using the TreeSelect component with checkbox selectionMode and found some strange behavior. 

If I have selected some checkboxes and want to deselect them by clicking on the click area of the checkbox, the overlay closes. I would expect the overlay to stay open, just as it stays open when I click elsewhere in the click area of the list item.

You can see this behavior [here](https://codesandbox.io/s/jdsjvy) at your official example.

Any tips on how I can change the behavior so that the overlay stays open when I click on the box? 

Thank you in advance!

**Comment by tarmaxz** [comment]
I'm also experiencing the same problem, even the example in the documentation has this behavior.

**Comment by mertsincan** [comment]
Interesting! This seems a core issue. Could you please create a Github issue with a sample stackblitz link to replicate it? For now, closed.
